### PR TITLE
New serverless pattern- ecs-windows-cdk-dotnet

### PR DESCRIPTION
**Pull request by PankajRawat333**
*Issue #, if available:*

*Description of changes:* This AWS CDK application demonstrates how to deploy a application on Windows based Amazon Elastic Container Service (Amazon ECS) Cluster.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: ecs-windows-cdk-dotnet/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Deployment Instructions
+
+1. Clone the project to your local working directory.
+    ```
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change the working directory to this pattern's directory.
+    ```
+    cd ecs-windows-cdk-dotnet/cdk/src
```
Comment (mavi888) [inline ecs-windows-cdk-dotnet/README.md:30]: For me to build work this needs to happen inside the WindowsECS folder.

File: ecs-windows-cdk-dotnet/cdk/src/WindowsECS/WindowsECS.csproj [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <!-- Roll forward to future major versions of the netcoreapp as needed -->
+    <RollForward>Major</RollForward>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- CDK Construct Library dependencies -->
+    <PackageReference Include="Amazon.CDK" Version="1.197.0" />
```
Comment (mavi888) [inline ecs-windows-cdk-dotnet/cdk/src/WindowsECS/WindowsECS.csproj:12]: This is using AWS CDK v1 please update to use V2

File: ecs-windows-cdk-dotnet/README.md (hunk 1 above)
Comment (PankajRawat333) [reply ecs-windows-cdk-dotnet/README.md:30]: @mavi888 I found an issue with my solution file, now I have fixed and pushed the changes, it should work without going inside WindowsECS folder.

**Comment by mavi888** [comment]
Thanks for submitting this pattern. 
You can find it here: https://serverlessland.com/patterns/ecs-windows-cdk-dotnet

Cheers

**Pull request merged**